common-android/src/main/AndroidManifest.xml

```diff
@@ -2,5 +2,4 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.amplitude.common.android">
 
-    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
```

Comment:
We only need to include the user-permission in the application, right?

Reply:
That's correct.